src/test/java/pl/danyboy/PasswordManagerTest.java

```diff
@@ -0,0 +1,29 @@
+package pl.danyboy;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PasswordManagerTest {
+
+    @Test
+    public void shouldAddPasswordWhenIsUnique() {
+        PasswordManager passwordManager = new PasswordManager();
+        String password = "Apetyt rośnie w miarę jedzenia";
+
+        passwordManager.setPasswords(password);
+
+        assertEquals(password, passwordManager.getRandomPassword());
```

Comment:
Tutaj pokusiłbym się o nieco bardziej rozbudowany test case. Albo dodał kolejny. Bo co tu sprawdzamy to czy dodanie i losowanie jednego hasła jest ok (czyli np. checkValues przejdzie) i to jest jak najbardziej ok. Natomiast nie ma sprawdzania elementu losowego. Można dodać kolejne test case, dodac tam dwa hasła, wylosować dwa.

```
String password1 = "Apetyt rośnie w miarę jedzenia";
String password2 = "Second one";
String randomPassword1 = passwordManager.getRandomPassword();
String randomPassword2 = passwordManager.getRandomPassword();
```

Potem spradzić w teście, że jeśli pierwsze haslo wylsowane to password1 to assertem sprawdzic czy drugie wylosowane to password2, albo jeśli pierwsze wylosowane to pass2 to wóczas drugie wylosowane powinno być pass1

```
if(randomPassword1.equals(password1)) { assertEquals(randomPassword2, password2) }
esle if(randomPasswor1.equals(password2)) { assertEquals(randomPassword2, password1) }
else { fails("whoops")
```